components/search/NotFound.tsx

```diff
@@ -0,0 +1,158 @@
+import type { ImageWidget } from "apps/admin/widgets.ts";
+import Image from "apps/website/components/Image.tsx";
+import SearchedWord from "$store/islands/SearchedWord.tsx";
+import { Picture, Source } from "https://denopkg.com/deco-cx/apps@0.25.13/website/components/Picture.tsx";
+import ProductShelf from "$store/components/product/ProductShelf.tsx";
+import type { Props } from "$store/components/product/ProductShelf.tsx"
+
+export interface PropsNotFound {
+    title: string;
+    imagem?: {
+        active?: boolean;
+        src: ImageWidget;
```

Comment:
tem que ser opcional
```suggestion
        src?: ImageWidget;
```